docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md

```diff
@@ -14,42 +14,46 @@ ms.author: "ronpet"
 ---
 # How to: Install an assembly into the global assembly cache
 
-There are two ways to install a strong-named assembly into the global assembly cache (GAC).
+The global assembly cache (GAC) stores assemblies that several applications share. Install an assembly into the [global assembly cache](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/gac) with one of the following components: 
+- [Windows Installer](#windows-installer)
+- [Global assembly cache tool](#global-assembly-cache-tool).
 
 > [!IMPORTANT]
-> Only strong-named assemblies can be installed into the GAC. For information about how to create a strong-named assembly, see [How to: Sign an Assembly with a Strong Name](how-to-sign-an-assembly-with-a-strong-name.md).
+> You can install only strong-named assemblies into the GAC. For information about how to create a strong-named assembly, see [How to: Sign an assembly with a strong name](how-to-sign-an-assembly-with-a-strong-name.md).
 
 ## Windows Installer
 
-[Windows Installer](/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache and other benefits. You can use the [WiX Toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension) to create an installer package for Windows Installer.
+[Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache and other benefits. To create an installer package for Windows Installer, use the [WiX toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension).
 
 ## Global assembly cache tool
 
-You can use the [Global assembly cache tool (gacutil.exe)](../tools/gacutil-exe-gac-tool.md) to add strong-named assemblies to the global assembly cache and to view the contents of the global assembly cache.
+You can use the [global assembly cache tool (gacutil.exe)](../tools/gacutil-exe-gac-tool.md) to add assemblies to the global assembly cache and to view the contents of the global assembly cache.
 
    > [!NOTE]
-   > Gacutil.exe is only for development purposes and should not be used to install production assemblies into the global assembly cache.
+   > *Gacutil.exe* is for development purposes only. Don't use it to install production assemblies into the global assembly cache.
 
-The syntax for gacutil is as follows:
+The syntax for *gacutil.exe* is as follows:
```

Comment:
```suggestion
The syntax for using *gacutil.exe* to install an assembly in the GAC is as follows:
```
I think it's best to make it clear that this is only one of the syntactical variants.